src/main/java/hudson/plugins/helpers/BuildProxy.java

```diff
@@ -100,15 +105,19 @@ public static boolean doPerform(Ghostwriter ghostwriter,
     private static RuntimeException unwrapException(Exception e,
                                                     BuildListener listener)
             throws IOException, InterruptedException {
+
         if (e.getCause() instanceof IOException) {
-            throw new IOException2(e.getCause().getMessage(), e);
+            throw new IOException(e.getCause().getMessage(), e);
         }
         if (e.getCause() instanceof InterruptedException) {
             e.getCause().printStackTrace(listener.getLogger());
             throw new InterruptedException(e.getCause().getMessage());
         }
         if (e.getCause() instanceof RuntimeException) {
-            throw new RuntimeException(e.getCause());
+            RuntimeException ex = new RuntimeException(e.getCause());
+            // It is required to triage JEP-200 security exceptions
+            ex.addSuppressed(e);
+            throw ex;

```

Comment:
Weird. Why not simply

```java
throw new RuntimeException(e);
```

?

Reply:
Tried to follow the original logic with which I generally disagree (see the comment above)